src/fluree/db/virtual_graph/bm25/index.clj

```diff
@@ -257,29 +256,27 @@
   (-match-class [_ _fuel-tracker _solution _s-mch _error-ch]
     where/nil-channel)
 
-  (-activate-alias [this _]
```

Comment:
I don't think it's a good idea to have partial protocol implementations. In this case, we should probably return an empty channel if the alias the caller is trying to activate doesn't match the alias of this graph.